### DISCUSSION TITLE
Checkbox for options

### DISCUSSION THREAD

**Discussion by balane** [cdmoro/bootstrap-vue-3#1518] [Q&A]
What would be the best/easiest way to add a checkbox to indicate an option is selected? Or is it even possible?

**Comment by VividLemon** [comment]
What are you asking

**Reply by balane** [reply]
Thanks for the reply but I got it working. If you're interested: I hadn't found a select/dropdown that had the bootstrap look and functionality we needed so I basically wrapped BDropdown and made it a select box with checkboxes to indicate an item is selected.
